### ISSUE TITLE
Implement on:paymentmethod and on:shippingaddresschange

### ISSUE THREAD

**Issue by yasserlens** [joshnuss/svelte-stripe#63]
### Describe the problem

I'm trying to figure out how to listen to shipping method changes to update the total, but can't figure this out yet.
There's a clear example on how to do this in Stripe's docs but no mention of it in svelte-stripe. It seems like the shpping method update event isn't propagated or added to the API. 

Here's a reference to it
https://stripe.com/docs/stripe-js/elements/payment-request-button?client=html#html-js-collecting-shipping-info

### Describe the proposed solution

- Add shipping method update events to the API if not yet there, OR
- If it's there, mention it in the documentation.

### Importance

I cannot use Svelte-Stripe without it

### Additional Information

I need this since we have different shipping methods with different pricing, and "express" payment is important to us.

**Comment by yasserlens** [comment]
FYI
I tried this
<PaymentRequestButton {paymentRequest} on:paymentmethod={handleExpressPayment} on:shippingaddresschange={handleShippingAddressChange} />

handleShippingAddressChange is never called on shipping method update. Not sure what else to explore to get that event bubble up to the PaymentRequestButton element?
Also, when the event is captured, would "event.updateWith(...)" or "event.detail.updateWith(...)" work to update the price or any other detail - basically to talk back to the payment module UI (google/apple/link..)?

**Comment by joshnuss** [comment]
Hi @yasserlens,

Thanks for reporting the issue.

I looked into it, and it seems that `<PaymentRequestButton>` is not yet exposing those events (`on:paymentmethod`, `on:shippingaddresschange`)

It shouldn't be too hard to correct, I will open a PR in the next few days.

If you'd like a quick workaround, the `element` is exposed as a prop, so you can bind to it and then handle the events in your higher level component:

```svelte
<!-- your payment page -->
<PaymentRequestButton bind:element={paymentRequestElement} {paymentRequest}/>

<script>
  let paymentRequestElement
  
  onMount(() => {
     paymentRequestElement.on('shippingaddresschange', ev => {
       // todo
    })
  })
</script>
```

Note: I haven't tested that, but I think it should work something like that.

**Comment by joshnuss** [comment]
@yasserlens The issue is now resolved in #80
